src/scripts/helpers/zotonic_setup

```diff
@@ -40,8 +40,7 @@ export -f require_zotonic_not_running
 function enablesite {
     config="priv/sites/$1/config"
     expr="s/enabled, \(true\|false\)/enabled, $2/"
-    perl -e "$expr" -pi $config
-    #echo sed -e "$expr" -i $config
+    sed -e "$expr" -i.bak -e $config
```

Comment:
Agreed, it might be a leftover from long ago, you have my blessing to remove it :)